ivadomed/config/config_dataset.json

```diff
@@ -0,0 +1,191 @@
+        "extensions": [".nii.gz"],
+        "roi_params": {
+            "suffix": null,
+            "slice_filter_roi": null
+        },
+        "contrast_params": {
+            "training_validation": ["T1w", "T2w", "T2star"],
+            "testing": ["T1w", "T2w", "T2star"],
+            "balance": {}
+        },
+        "slice_filter_params": {
+            "filter_empty_mask": false,
+            "filter_empty_input": true
+        },
+        "slice_axis": "axial",
+        "multichannel": false,
+        "soft_gt": false,
+        "bids_validate": true
+    },
+    "Datasets": [
+        {
+            "dataset_label": "DataSet1",
+            "training": [
+                {
+                    "type": "BIDS",
+                    "subset_label": "Subset1",
+                    "path_data": "/Path to/subset_folder_1/",
+                    "BIDS_ACQ": ["Sag"],
+                    "BIDS_RUN": ["1"],
+                    "BIDS_SESSION": [1],
+                    "BIDS_Contrast": ["T1w", ],
+                    "...":"...",
+                    "ground_truth": ["sub*_T1_labelled.nii"],
+                    "expected_input": 2,
+                    "expected_gt": 1,
+                    "missing_files_handle": "drop_subject",
+                    "excessive_files_handle": "use_first"
+                },
+                {
+                    "type": "REGEX",
+                    "subset_label": "Subset2",
+                    "path_data": "/Path to/subset_folder_2/",
+                    "image": ["/data/source1/.*/sub*_T1.nii", "/data/source1/.*/sub*_T2.nii"],
+                    "ground_truth": ["/data/source1/.*/sub*_T1.nii"],
+                    "expected_input": 2,
+                    "expected_gt": 1,
+                    "missing_files_handle": "drop_subject",
+                    "excessive_files_handle": "use_first_and_warn"
+                },
+                {
+                    "type": "FILES",
+                    "subset_label": "Subset3",
+                    "image_ground_truth": [
+                        [["/Path to/subset_folder_2/sub1_T1.nii", "/Path to/subset_folder_2/sub1_T2.nii"], ["/Path to/subset_folder_2/gt1"]],
+                        [["/Path to/subset_folder_2/sub2_T1.nii", "/Path to/subset_folder_2/sub1_T1.nii"], ["/Path to/subset_folder_2/gt2"]],
+                        [["/Path to/subset_folder_2/sub3_T1.nii", "/Path to/subset_folder_2/sub1_T1.nii"], ["/Path to/subset_folder_2/gt3"]]
+                    ],
+                    "expected_input": 2,
+                    "expected_gt": 1,
+                    "missing_files_handle": "drop_subject",
+                    "excessive_files_handle": "use_first_and_warn"
+
+                }
+            ],
+            "validation":[ Same Above Dict struct],
+            "test":[ Same Above Dict struct],
+            "training_validation":[ Same Above Dict struct],
+            "training_validation_test":[ Same Above Dict struct]
```

Comment:
A note to myself: maybe I'll learn something new, but check if this works or skipped while testing

Reply:
Yeah, my bad. I think that is just some example. I believe this JSON will be discarded in the PR update.